### PR TITLE
DCP00{0[0-9],1[0-2]}: Add voting results.

### PR DESCRIPTION
**Pull request by davecgh**
**This requires #31**.

This adds a new voting results subsection to the deployment section of all DCPs to document the historical blocks at which the voting started as well as the locked in and activation heights.

It also clarifies that implementations may optionally optimize their enforcement activation logic accordingly for those historical activation points as opposed to tallying historical votes in the applicable DCPs.

### CODE REVIEW

File: dcp-0012/dcp-0012.mediawiki [hunk 1]
```diff
@@ -203,6 +205,21 @@ voting infrastructure as follows:
...
+in and awaiting activation.
+
+{|
+!Status!!Block Hash!!Block Height
+|-
+|Voting Started||0000000000000000d608dc4ec7fcae5c650353db68a77f2954df74c25462f337||778240
+|-
+|Locked In||0000000000000000896042b2b0536a4046e56ef505f20f7301ca7e042a5c218e||786304
+|-
+|Active|TBD||794368
```
Comment (jholdstock) [inline dcp-0012/dcp-0012.mediawiki]:
```suggestion
|Active||TBD||794368
```

**Comment by jholdstock** [comment]
Maybe add this new table to the DCP-0000 template?

File: dcp-0012/dcp-0012.mediawiki (hunk 1 above)
Comment (davecgh) [reply dcp-0012/dcp-0012.mediawiki]: Nice catch.  Fixed.

**Review by jholdstock — APPROVED** [review]

**Comment by davecgh** [comment]
> Maybe add this new table to the DCP-0000 template?

Good idea.  Will update.

**Comment by davecgh** [comment]
> Maybe add this new table to the DCP-0000 template?

Good idea. Added.

**Review by jholdstock — APPROVED** [review]

**Pull request merged**